examples/ITK_Example18_MONAI_affine_Elastix_nonlinear.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.                        #scale_range=(-0.1, 0.1),
</pre><p>May this comment be removed, or does it suggest a future experiment or tweak? (Same for mode=nearest)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKElastix/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>